pkg/api/types.go

```diff
@@ -1041,6 +1041,95 @@ const (
 	DNSDefault DNSPolicy = "Default"
 )
 
+// A node selector represents the union of the results of one or more label queries
+// over a set of nodes; that is, it represents the OR of the selectors represented
+// by the nodeSelectorTerms.
+type NodeSelector struct {
+	// nodeSelectorTerms is a list of node selector terms. The terms are ORed.
```

Comment:
you can omit the JSON field name in all of the field descriptions. So for example this can just be "a list of node selector terms. The terms are ORed." I think that looks simpler. If you agree, please apply that suggestoin to all of the field comments.